plugins/woocommerce/client/admin/client/settings-payments/components/bank-accounts-list/bank-account-modal.tsx

```diff
@@ -41,22 +46,18 @@ export const BankAccountModal = ( {
 	onSave,
 	defaultCountry,
 }: Props ) => {
+	const countries = window.wcSettings.countries;
```

Comment:
Do we need those countries? I see that there is no countries in the old version. As far as I get, countries influence what fields are required, right? Could you link me please to the requirement from product/design about it, I seem to have missed it

Reply:
Yes, country selection is a requirement from the designs by Magda (I didn't add it in the previous PR because it was just easier to address it in this one) - here's the P2 post which outlines the requirements: pfHfG4-17j-p2